RELEASE_NOTES.rst

```diff
@@ -1,3 +1,52 @@
+coala 0.9.0
+===========
+
+Command Line Interface Changes:
+
+- Users can press enter to dismiss a result by default.
+- A lot more strings, like ``roger`` or ``no way`` are allowed for boolean
```

Comment:
maybe sort these by severity or something? This should move WAY to the bottom or nobody is taking the list serious from here on.